delegation/tasks/3-in-progress/ASK-0029-multi-evaluator-architecture.md

```
@@ -0,0 +1,526 @@
+# Verify evaluator works (requires OPENAI_API_KEY)
+adversarial fast-check delegation/tasks/2-todo/ASK-0029-multi-evaluator-architecture.md
+```
+
+### Edge Cases
+
+| Scenario | Expected Behavior |
+|----------|------------------|
+| Install without git | Exit with error, show install instructions |
+| Install with existing evaluators | Warn and skip (use `--force` to override) |
+| Clone timeout (>60s) | Exit with error, suggest network check |
+| Invalid --ref tag | Exit with error, show tags URL |
```

markdownlint-cli2 (0.20.0)

[warning] Horizontal rule style
Expected: ----; Actual: ---

(MD035, hr-style)